### PR TITLE
Blog post on integration test

### PR DESCRIPTION
**Pull request by scottcarol**
Adding blog post on front-end integration test, images and adding myself as a blog author.

### CODE REVIEW

File: linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+One of the coolest things about working on [Linkerd](http://www.linkerd.io) is
+how excited our users are about our clean, deceptively simple dashboard, built
+with [React](https://reactjs.org/) and [Material-UI](https://material-ui.com/).
+(Wanna try it? Get our super-light, open-source service mesh [up and running in
+just a few minutes](https://linkerd.io/2/getting-started/)!)
+
+{{< fig
+  alt="Screenshot of Linkerd dashboard"
+  src="/uploads/2019/04/linkerd-dashboard-screenshot.png" >}}
+*The default view of the Linkerd Dashboard, as of edge release 19.3.2.*
```
Comment (grampelberg) [inline linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: Why not have this as part of `fig`?

File: linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+
+And when I say excited, I mean unsolicited-praise excited: we constantly get
+messages from users like this:
+
+{{< fig
+  alt="Screenshot of Linkerd dashboard"
+  src="/uploads/2019/04/happy-tweet.png" >}}
+*Tweet from a happy user: "Very pleased with #Linkerd2 - deployed my app (with
```
Comment (grampelberg) [inline linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: Probably don't need this at all.

File: linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+- [Yarn](https://yarnpkg.com/en/docs/getting-started)
+- [Node.js](https://nodejs.org/en/download/)
+
+**Recommended**: If you install Yarn through the [Homebrew package
+manager](https://brew.sh/) it will download Node.js as well.
+
+First, navigate to the directory in your project that holds your web
+application. Ours is named `app`:
+
+    cd app
```
Comment (grampelberg) [inline linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: Can you do fences for these (like the other docs/blosts)?

File: linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+`devDependencies` portion of our `package.json`, since we'll be running our
+tests in a development environment. If you didn't already have a `package.json`
+file, this command will create one for you.
+
+Now, we'll add WebdriverIO's testrunner command line interface, which is what
+will allow you to run the tests from your terminal:
+
+    yarn add @wdio/cli --dev
+
+**Note:** When WebdriverIO 5 was released, the project switched to a `@wdio`
```
Comment (grampelberg) [inline linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: There's a shortcode for this `{{< note >}}`.

Comment (scottcarol) [reply linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: Thanks, TIL.

**Comment by scottcarol** [comment]
Hey @grampelberg, thanks for the feedback. I took all of your suggestions, except I left the text of the tweet spelled out for accessibility reasons. 

I tried to put the tweet text into a multi-line `alt` while still passing linting rules:

```
{{< fig
  alt="Text of tweet by happy user: Very pleased with #Linkerd2 - deployed my
  app (with auto-proxy-injection) and #itjustworked - Had all the info I needed
  on the dashboard - Thanks very much (great docs too)"
  title="Tweet by a happy Linkerd user!"
  src="/uploads/2019/04/happy-tweet.png" >}}
```

but this broke the string and the page wouldn't render. Any ideas on how to fix?

**Comment by grampelberg** [comment]
@scottcarol I'd just get rid of the tweet text on that one. Maybe the author's twitter account instead?

**Comment by grampelberg** [comment]
@scottcarol I'd just get rid of the tweet text on that one. Maybe the author's twitter account instead?

**Comment by grampelberg** [comment]
@scottcarol I'd just get rid of the tweet text on that one. Maybe the author's twitter account instead?

**Comment by grampelberg** [comment]
@scottcarol I'd just get rid of the tweet text on that one. Maybe the author's twitter account instead?

**Comment by grampelberg** [comment]
@scottcarol I'd just get rid of the tweet text on that one. Maybe the author's twitter account instead?

**Comment by grampelberg** [comment]
@scottcarol I'd just get rid of the tweet text on that one. Maybe the author's twitter account instead?

**Comment by scottcarol** [comment]
@grampelberg I think we should keep the text - a screenreader user won't be able to read the text of the screenshotted tweet, and the content is important to include because it tells the story of why users love Linkerd. 

[A11y nonprofit WebAim](https://webaim.org/techniques/alttext/) recommends that if an image contains text, that text should be displayed as `alt` text - since a multi-line `alt` tag doesn't seem possible here, I'd like to leave it in the post. 

Linking to the author's twitter risks him changing or quitting his account, and/or removing the tweet.

**Comment by grampelberg** [comment]
Makes sense, just throw a disable for the linter around it. I'm not sure I have a better solution than that off the top of my head.

**Comment by scottcarol** [comment]
Ok @grampelberg, should be fixed now!

File: linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+`sauceConnect` and added two specific browsers to try, Firefox and Chrome.
+
+# Tunneling to Localhost in 3, 2, 1
+
+[SauceConnect](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy) is a
+proxy server that will open a tunnel between a SauceLabs Virtual Machine (VM)
+and whatever you are running on `localhost.`You'll want to save it in a separate
+directory from the rest of your development files.
+
+**Note About Port Numbers:**  SauceConnect only [supports specific port numbers
```
Comment (grampelberg) [inline linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: Did you want to `{{< note >}}` here?

Comment (scottcarol) [reply linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: Nooo I thought I fixed that! Let me do that right now

Comment (scottcarol) [reply linkerd.io/content/blog/browser-testing-from-scratch-building-quick-and-easy-integration-tests-with-webdriverio-and-saucelabs.md]: Ok done, thanks for the catch on that one!

**Pull request merged**